docs/using-workflows/adding-secrets.md

```diff
@@ -38,4 +44,22 @@ steps:
     nodeCount: 3
     zones: ["us-west1-a"]
 ```
+## About Secrets
+Secrets are encrypted when entered and remain encrypted until they are used by a workflow.
```

Comment:
This section is almost identical to the connections section ; maybe they should just be together in one doc that points out the differences?